### ISSUE TITLE
知识蒸馏 | Measure Zero

### ISSUE THREAD

**Issue by Shiina18** [Shiina18/shiina18.github.io#106]
https://shiina18.github.io/machine%20learning/2021/08/05/knowledge-distill/ 

 模型压缩, 用小模型向大模型学习. 神经网络最后一层通常将 logits $z_i$ 经过 softmax 函数转化为类别概率预测 $q_i$ 输出, \[q_i = \frac{\exp(z_i/T)}{\sum_j \exp(z_j/T)},\] 其中 $T=1$. 这种概率预测称为 soft target/label (相对于 hard target, 即直接预测一个类别, 而不给与概率).
